src/api/pastel-rpc/rpc.ts

```diff
@@ -8,7 +9,7 @@ export type TRPCConfig = {
 
 export async function rpc<T>(
   method: string,
-  params: string[],
+  params: any[],
```

Comment:
This should not change. Params are always a list of strings

Reply:
I have updated the params